src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import Navbar from '../../components/Navbar/Navbar'
+import Banner from '../../components/Banner/Banner'
+import { useCart } from '../../context/cart-context'
+import './Home.css';
+import Footer from '../../components/Footer/Footer'
+
+const Home = () => {
+    const { dispatch } = useCart();
```

Comment:
you may remove `dispatch` as it isn't used in the file

Reply:
done